conn.go

```diff
@@ -879,11 +903,17 @@ func (c *Conn) exec(ctx context.Context, req frameWriter, tracer Tracer) (*frame
 	// resp is basically a waiting semaphore protecting the framer
 	framer := newFramerWithExts(c, c, c.compressor, c.version, c.cqlProtoExts)
 
+	var streamContext StreamObserverContext
+	if c.streamObserver != nil {
+		streamContext = c.streamObserver.StreamContext(ctx)
```

Comment:
I think we can avoid allocating another variable if we move this block after `call` variable initialization below. Smth like:

```go
	call := &callReq{
		framer:   framer,
		timeout:  make(chan struct{}),
		streamID: stream,
		resp:     make(chan error),
	}
	if c.streamObserver != nil {
		call.streamObserverContext = c.streamObserver.StreamContext(ctx)
	}
```

Reply:
Moved, thanks!